db/models/topics-models.js

```diff
@@ -1,5 +1,12 @@
 const db = require("../connection")
+const fs = require("fs/promises")
 
+const fetchEndpoints = () => {
+    return fs.readFile(`endpoints.json`, 'utf8', (err, data) => {return data})
+    .then((data) => {
+        return JSON.parse(data)
```

Comment:
This is OK, but as we've got endpoint.json in this repo, no need to use promises, could just require the file in. When you require, it'll be imported as an object.